app/views/shared/_post.html.erb

```diff
@@ -0,0 +1,5 @@
+<div>
+  <h2>Post #<%= post.id %></h2>
+  <p><%= post.text %></p>
+  <span>Comments: <%= post.comments_counter%>, Likes: <%= post.likes_counter%></span>
```

Comment:
- [ ] Please add a link so that leads to the show page for the post when a user clicks it. At this moment, the only way to get to the post show page is by manually typing in the URL.